service/service.go

```diff
@@ -130,7 +130,7 @@ func New(config Config) (*Service, error) {
 			CAFile:           config.Viper.GetString(config.Flag.Service.Etcd.TLS.CAFile),
 			CrtFile:          config.Viper.GetString(config.Flag.Service.Etcd.TLS.CrtFile),
 			CRDLabelSelector: config.Viper.GetString(config.Flag.Service.CRD.LabelSelector),
-			EtcdEndpoint:     config.Viper.GetString(config.Flag.Service.Etcd.Endpoint),
+			EtcdEndpoints:    config.Viper.GetStringSlice(config.Flag.Service.Etcd.Endpoints),
```

Comment:
Are you sure this works? How has the input have to look like to get a string slice in return?

Reply:
Fixed. 
String slice should look like `https://url1:2379 https://url2:2379`